shell/plugins/steve/subscribe.js

```diff
@@ -524,10 +593,19 @@ export const actions = {
   },
 
   'ws.resource.remove'(ctx, msg) {
-    queueChange(ctx, msg, false, 'Remove');
-
     const data = msg.data;
     const type = data.type;
+
+    if (type === 'schema') {
```

Comment:
References to `schema` need a bit of `import { SCHEMA } from '@shell/config/types';` lovin'